docs/tutorial/getting_started_with_model.rst

```diff
@@ -0,0 +1,156 @@
+Getting started with the Model class
+====================================
+
+This tutorial covers opening an existing RESQML dataset and identifying the high level objects contained within it.
+
+Contents
+1 Prerequisites
+2 Importing the model module
+3 Opening an existing RESQML dataset
+4 Keys to the RESQML high level objects
+5 Selectively listing high level objects
+6 Finding a single high level object
+7 Other methods in the Model class
+8 Summary
```

Comment:
The "contents" directive may be more appropriate here:

https://stackoverflow.com/a/24160594

Reply:
oh yeah I didn't bother including the contents from many of the pages - I can put them in or we can leave people to use the sidebar?